### ISSUE TITLE
Request: Epics, Types, and Priority

### ISSUE THREAD

**Issue by Velkas** [BaldissaraMatheus/Tasks.md#104]
Epics for grouping tasks, Types for grouping/sub-grouping inside epics, Priority for sorting.

Obviously tags can be used for this but these as separate metadata on tasks would help me organize, filter, and sort better without cluttering tags.

If there's more info required, please ask. Not sure what to include here as these are common kanban features.

**Comment by BaldissaraMatheus** [comment]
Could you clarify the request for Types? A concrete example should be useful

**Comment by Velkas** [comment]
Example Types could be:

- Bug
- Task
- Documentation
- Test
- Idea
- Feature

Given the structure of the project, this could just be a simple line-item list. Any items in that list will show up as items to select one from in a combobox when editing a card.

Also thanks for adding the label, I wasn't able to do that (repo perm issue?)

**Comment by BaldissaraMatheus** [comment]
> Also thanks for adding the label, I wasn't able to do that (repo perm issue?)
Yeah it's some issue on the repo, I need to take some time to see if there's something misconfigured
